soapui/src/main/java/com/eviware/soapui/SoapUI.java

```diff
@@ -788,6 +788,7 @@ public void run() {
         public void run() {
             boolean isDebug = java.lang.management.ManagementFactory.getRuntimeMXBean().
                     getInputArguments().toString().indexOf("-agentlib:jdwp") > 0;
+            StandaloneSoapUICore standaloneSoapUICore = new StandaloneSoapUICore(true);
             SoapUIUpdateProvider updateProvider = SoapUIAutoUpdaterUtils.getProvider();
             if (!isDebug && SoapUI.getSettings().getBoolean(VersionUpdateSettings.AUTO_CHECK_VERSION_UPDATE)) {
```

Comment:
here on "SoapUI.getSettings()... " defaultSoapUICore will be created and later changed on StandaloneSoapUICore.
Both these items will initialize setting file and all plugins.